doc/build-instructions/Build_Instructions_V11.md

```diff
@@ -730,7 +728,9 @@ bash configure --openjdk-target=${OPENJ9_CC_PREFIX} \
                --disable-warnings-as-errors
 ```
 
-:pencil: **Non-compressed references support:** If you require a heap size greater than 57GB, enable a noncompressedrefs build with the `--with-noncompressedrefs` option during this step.
+:pencil: **Non-compressed references support:** If you require a heap size greater than 57GB, enable a noncompressedrefs build with the `--with-noncompressedrefs` option during this step.  You need this option to use the JIT compiler.
```

Comment:
To make this clear, I think you should say:
"You need this option to use the JIT compiler regardless of heap size:  the JIT does not support compressed references yet."

Reply:
Reflected the review comment.